src/App.js

```diff
@@ -1,24 +1,56 @@
 import './App.css';
-import { useEffect } from 'react'
-// import axios from 'axios';
-
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import Header from './Header';
+import Explanation from './Explanation';
+import ApiData from './ApiData';
+import Footer from './Footer';
+import GetExercise from './GetExercise';
+import AddExercise from './AddExercise';
+import FormDropdown from './FormDropdown';
 
 function App() {
-  // useEffect(() => {
-  //   axios({
-  //     url: "https:/api.goprogram.ai/inspiration",
-  //     method: "GET",
-  //     dataResponse: "json"
-  //   }).then((response) =>{
-  //     console.log(response);
-  //   })
-  // })
-
+  const [quote, setQuote] = useState([])
+  useEffect(() => {
+    axios({
+      url: "https:/api.goprogram.ai/inspiration",
+      method: "GET",
+      dataResponse: "json"
+    }).then((response) =>{
```

Comment:
Try this to simplify the call and change quote state to an object at default to store response.data directly and in ApiData, you can get author and quote from the quote object:   

useEffect(() => {
    axios({
      url: 'https://api.goprogram.ai/inspiration',
      method: 'GET',
      dataResponse: 'json',
    })
      .then((response) => {
        setQuote(response.data); 
      })
      .catch((err) => {
        alert('Something went wrong with the API Quote, sorry for the inconvenience!', err);
      });
  }, []);